fix build of other packages (e.g. py-scipy) in some cases (e.g. when
WRKOBJDIR has a trailing slash)

Index: numpy/distutils/command/build_src.py
--- numpy/distutils/command/build_src.py.orig
+++ numpy/distutils/command/build_src.py
@@ -370,6 +370,7 @@ class build_src(build_ext.build_ext):
             #    incl_dirs.append(self.build_src)
             build_dir = os.path.join(*([self.build_src]\
                                        +name.split('.')[:-1]))
+        build_dir = os.path.normpath(build_dir)
         self.mkpath(build_dir)
         for func in func_sources:
             source = func(extension, build_dir)
